### DISCUSSION TITLE
Z Probing not working out of the box after uploading

### DISCUSSION THREAD

**Discussion by techboycr** [luc-github/Repetier-Firmware-0.92#386] [Q&A]
I just did a fresh install form the latest git, but for some reason G29 is just not working at all. Always show the error Probe Failed.

Da Vinvi 1.0 with repetier host
Compiled from Arduino

Thanks in advance!

**Comment by luc-github** [comment]
@techboycr sorry just noticed this issue
Did you manual bed leveling after flashing as requested in installation steps ?
